docs/serverless-v2/01-02-serverless-v2.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Architecture
+---
+
+Serverless v2 relies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions, and [Tekton](https://github.com/tektoncd/pipeline) as a pipeline for creating Docker images. See how these and other resources process a lambda within a Kyma cluster:
+
+![Serverless v2 architecture](./assets/serverless-v2-architecture.svg)
+
+1. You create a lambda either through the UI or by applying a Function custom resource (CR). This CR contains the lambda definition (business logic that you want to execute) and information on the environment on which it should run (Node 6 or Node 8).
```

Comment:
`... (at the moment we only support NodeJS 6 and NodeJS 8)` But better will be `Note` with this information.